src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandler.java

```diff
@@ -160,6 +161,20 @@ protected PreSubmitCallbackResponse<SscsCaseData> issueDocument(Callback<SscsCas
         return response;
     }
 
+    protected String getDocumentTypeLabel(SscsCaseData caseData, DocumentType documentType, String documentTypeLabel) {
+
+        String embeddedDocumentTypeLabel = (FINAL_DECISION_NOTICE.equals(documentType) ? "Decision Notice" : documentTypeLabel);
+
+        PostHearing postHearing = caseData.getPostHearing();
+        if (nonNull(postHearing) && nonNull(postHearing.getSetAside().getAction())) {
```

Comment:
1) This will throw a NPE if setAside is null
2) Does it require a feature flag check as well?

Reply:
SetAside is always not null. feature flag check is not required.